rico.md

```diff
@@ -0,0 +1,69 @@
+
+Rico originates from the [_Dolphin Platform_](https://github.com/canoo/dolphin-platform/) which appears to be discontinued. Rico is an (originally) API compatible fork that we continue to develop and improve.
+
+### Rationale
+
+The motivation for us to create Rico is to avoid code for standard use-cases to be written over-and-over again. Thus, we created it as an extension to commonly used frameworks like Spring and Jakarta EE (Java EE) that is easily adoptable.
+
+The outcome is a framework that is build on top of well-known frameworks combined with commonly used components and prepared to be used with a very short warmup. 
+Rico helps you to build your applications in a short time letting you focus on the business requirements as it has standard solutions for common technical use cases like _Data Access_, _Logging_, _Monitoring_ and _Security_ *already build in* and well prepared to get *integrated in your environment*.
+
+To achieve this, Rico provides clean APIs to seperate complexity coming from the set of technologies required to build your solution from your actual code required to provide your business solution.
+
+Also, Rico provides a remoting layer that helps you to seperate UI framework specific code so that is a lot easier to switch UI implementations.
+It also enables you to choose different technologies for every target platform. 
+As we see a very rapidly changing environment in UI technologies, this seperation helps to protect your investment put into the solution build with Rico as it will be less dependent on a concrete UI technology.
+
+### Getting started
+
+Currently, the best point to get started is our [samples project repository](https://github.com/rico-projects/rico-samples). Please refer to the README in each project for a description how to get started with the sample:
+
+* [Rico To-Do list application](https://github.com/rico-projects/rico-samples/tree/master/todo-list) Application both for Spring and Jave EE with frontends in Angular, AngularJS, Polymer and JavaFx to demonstrate _Rico Remoting_ and an _EventBus_ to connect between different clients implemented in various UI technolgies.
+* [Rico HTTP rest security](https://github.com/rico-projects/rico-samples/tree/master/rest-security) - use Rico integration for Keycloak to secure your applications.
+* [Rico HTTP client](https://github.com/rico-projects/rico-samples/tree/master/http-client) - show usage of Rico HTTP client to consume web services over HTTP 
+
+Our main repository as well as all our subprojects can be found at our [GitHub organisation](https://github.com/rico-projects/).
+
+## Who is using Rico
+
+Users of Rico we know of and who have agreed to be listed here as references:
+
+* [Sprouts GmbH](https://www.sprouts.aero/)
+* [IRIX Software Engineering AG](https://www.irix.ch/)
+
+If you are using Rico and would be happy to see your company listed here, please [contact us](mailto:info@karakun.com).
```

Comment:
maybe a separate email address?

Reply:
already answered in the other discussion about this :)